### PR TITLE
Update everything to use container|serverice name semantics

### PR DESCRIPTION
**Pull request by hopsoft**

### CODE REVIEW

File: lib/containers/commands/list.rb [hunk 1]
```diff
@@ -5,13 +5,13 @@ class Containers::CLI < Thor
   method_option :detailed, type: :boolean, aliases: "-d", desc: "List detailed container information"
   method_option :service, type: :boolean, aliases: "-s", desc: "List container service names"
   def list(*args)
-    return execute_command "docker ps -a | grep #{project_name}" if options[:detailed]
+    return execute_command "docker ps -a | grep #{app_name}" if options[:detailed]
 
     if options[:service]
       command = "containers list"
-      puts_command "#{command} | #{Rainbow("(strip project name)").green.faint}"
+      puts_command "#{command} | #{Rainbow("(strip app name)").green.faint}"
```
Comment (coderberry) [inline lib/containers/commands/list.rb]: Should this be `app_name` instead of `app name`?

File: lib/containers/commands/tail.rb [hunk 2]
```diff
@@ -6,6 +6,6 @@ class Containers::CLI < Thor
   method_option :service, type: :array, aliases: "-s", desc: "A list of service names (space delimited)"
   def tail(*args)
     args << "--since 5m" unless args.include?("--since")
-    execute_command "docker compose logs #{args.join " "} -f #{requested_service_names.join " "}"
+    execute_command "docker compose #{docker_compose_files.map { |f| "-f #{f}" }.join " "} logs #{args.join " "} -f #{requested_service_names.join " "}"
```
Comment (coderberry) [inline lib/containers/commands/tail.rb:9]: This seems to be used a lot. I wonder if it should be added to a base class or concern?

```ruby
docker_compose_files.map { |f| "-f #{f}" }.join " "}
```

**Review by coderberry — APPROVED** [review]
Looks great bro!

File: lib/containers/commands/tail.rb (hunk 2 above)
Comment (hopsoft) [reply lib/containers/commands/tail.rb:9]: Definitely could be refactored but I'll probably wait on that.

**Pull request merged**